### PR TITLE
feat(second_page): support navigation in push and fix Android

### PR DESCRIPTION
**Pull request by Santi92**
This PR

* New page `SecondPage` for test navigation when arrived push.
* Fix Android push when app is background

### CODE REVIEW

File: analysis_options.yaml [hunk 1]
```diff
@@ -1 +1 @@
-include: package:lint/analysis_options_package.yaml
+
```
Comment (cdmunoz) [inline analysis_options.yaml:3]: I think that is important to leave a comment explaining why the include is set here. Probably adding the link to the workaround as well. WDYT?

**Review by cdmunoz — CHANGES_REQUESTED** [review]
Excellent! 🥇 

I left a small comment ☺️

File: analysis_options.yaml (hunk 1 above)
Comment (Santi92) [reply analysis_options.yaml:3]: Done!

**Review by cdmunoz — APPROVED** [review]
Osom! 💪🏻

**Pull request merged**